### PR TITLE
Fix event rule operation ID

### PR DESCRIPTION
**Pull request by ashkarpetin**

### CODE REVIEW

**Review by p3yman — APPROVED** [review]

File: openapi/paths/events@{eventType}@rules.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ put:
   tags:
     - Rules
   summary: Update the rules for event
-  operationId: PutEventRuleCollection
+  operationId: PutEventRule
```
Comment (slavcodev) [inline openapi/paths/events@{eventType}@rules.yaml:37]: I think this might be confusing, since the operation works with rules set and not one single rule.

Comment (ashkarpetin) [reply openapi/paths/events@{eventType}@rules.yaml:37]: @2e3s Please check this

https://github.com/Rebilly/api-definitions/pull/364/files#diff-c0ec58b81489a48627f8f1c6b4f570c7d70ac5265736e3dbbddf58dcd4698123R59

Comment (ashkarpetin) [reply openapi/paths/events@{eventType}@rules.yaml:37]: @slavcodev Are you suggesting to leave `PutEventRuleCollection`?

**Review by slavcodev — APPROVED** [review]

File: openapi/paths/events@{eventType}@rules.yaml (hunk 1 above)
Comment (2e3s) [reply openapi/paths/events@{eventType}@rules.yaml:37]: I don't see how it's inconsistent and affects my PR

**Pull request merged**